README.md

```diff
@@ -14,19 +14,19 @@ To install pyAudio on windows use:
 for other platform
 > pip install pyaudio
 
-- **Kivy:** [https://kivy.org/#home](https://kivy.org/#home)
+- **Kivy:** <https://kivy.org/#home>
 
-- **KivyMD:** [https://github.com/HeaTTheatR/KivyMD](KivyMD, Version 0.100.2)
+- **KivyMD:** [KivyMD, Version 0.100.2](https://github.com/HeaTTheatR/KivyMD)
```

Comment:
This was the broken link that caught my attention, the rendered readme was completely off